firmware/controllers/gauges/tachometer.cpp

```diff
@@ -9,47 +9,118 @@
  * @date Aug 18, 2015
  * @author Andrey Belomutskiy, (c) 2012-2018
  */
-
+//#include "engine.h"
```

Comment:
why dead code? if you want to remove it - just remove it?

Reply:
Still in work, mck1117 helped me to start with the unit_tests, it is not the final code here